index.js

```diff
@@ -27,7 +33,9 @@ module.exports = {
 	// Get all values starting with a certain pathname and filter their views
   getAll: async function getAll(options) {
     const data = {}
-    const keys = (await module.exports.keys()).filter(key => key.startsWith(options.pathname))
+    const keys = (await module.exports.keys()).filter((key) => {
+      return options.wildcard ? key.match(keyRegex(options.pathname)) : key.startsWith(options.pathname)
```

Comment:
This won't work I'm pretty sure, since `options.wildcard` comes from the query param it'll be `"false"`. (also this should be opt-out I thought, not opt-in)